core/modules/redirect/redirect.admin.inc

```diff
@@ -579,6 +579,7 @@ function _redirect_extract_url_options(&$element, &$form_state) {
 function redirect_edit_form_validate($form, &$form_state) {
   form_state_values_clean($form_state);
   $values = $form_state['values'];
+  // Get rid of form values not appropriate for Redirect creation.
```

Comment:
Is the word "not appropriate" the best to describe things here, or would "not required" be better?

Also, can we please add a note to say that this change was needed, otherwise errors are thrown in php 8.2+.

Reply:
Wording... :shrug:. I'm open for suggestions. :wink:  Personally, I thought that "not appropriate" fits best for form item submissions, that have nothing to do with the actual Redirect object.